common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -1622,6 +1622,126 @@ public void numPointsUnsupported() throws Exception {
     assertEquals(expected, e.getMessage());
   }
 
+  @Test
```

Comment:
I would suggest we break down this unit test into multiple ones while each of them test one geometry type. Note that it would highlight the difference between multi-point and other types where ordering matters.

Reply:
Done. One more thing, this implementation keeps the ordering of the coordinates in MultiPoint.